### PR TITLE
Remove DS store

### PR DESCRIPTION
**Pull request by gracicot**
WARNING: macOS detected

The PR adds all hidden files in the gitignore and delete da store from the repo

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,7 +4,12 @@
...
-
 npm-debug.log*
+
+# hidden files
+.*
+
+# whitelist hidden files
+!.gitignore
+!.prettierrc
+!.husky
```
Comment (Alex-Hayeur) [inline .gitignore:15]: husky
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣀⣀⣠⣤⣤⣤⣤⣀⣀⣀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣠⣤⡶⠟⠛⠋⠉⠉⠉⠁⠀⠀⠀⠉⠉⠙⠛⠻⠶⢦⣤⣀⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⣴⠞⠋⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠙⠛⢷⣤⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣤⠾⠋⠀⢠⣄⣀⣤⣤⣤⣤⣤⣄⠀⠀⠀⠀⠀⠀⠀⢠⣤⣤⣤⣤⣤⣀⣀⡀⠀⠙⠻⣶⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣴⠟⠁⠀⠀⠀⣿⣿⣿⣿⣿⠿⠿⠿⠛⠀⠀⠀⠀⠀⠀⠀⠈⠛⠿⠿⣿⣿⣿⣿⣿⠀⠀⠀⠈⠻⣦⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⠟⠁⠀⠀⠀⠀⠀⠈⠉⠉⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠉⠉⠟⠿⠀⠀⠀⠀⠀⠈⠻⣆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣼⠏⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⠀⠀⠀⠀⠀⠀⠀⠀⠙⣷⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⢀⣾⠃⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠳⣄⠀⠀⠀⠀⠀⠀⠀⠘⣷⡀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⣼⠃⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣀⣀⣠⣤⣄⣀⣀⣀⡀⠀⠀⠀⠀⠀⣀⣀⣀⣀⣀⣠⣤⣀⣀⣀⠙⣧⠀⠀⠀⠀⠀⠀⠰⡜⣷⡄⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⢸⡏⠀⠀⠀⠀⠀⠀⠀⢀⣴⡿⠛⠉⠉⠈⣭⣿⣿⣿⣯⡙⠛⢶⣦⡶⠛⠛⣿⣿⣿⣿⣏⠀⠀⠉⠉⠻⣿⡆⠀⠀⠀⠀⠀⠀⣿⣿⣷⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⣿⠁⠀⠀⠀⠀⠀⠀⠀⣼⣿⠀⠀⠀⠀⠀⣿⣿⣿⣿⣿⠇⠀⢀⣿⠀⠀⠸⣿⣿⣿⣿⣿⠃⠀⠀⠀⢀⣾⣷⡆⠀⠀⠀⠀⠀⢸⣿⣿⡇⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⢰⣿⠀⠀⠀⠀⠀⠀⠀⠀⢷⡛⣧⡀⠀⠀⠀⠘⠻⡿⠿⠋⠀⠀⣼⠿⣧⡀⠀⠙⠻⠿⠿⠋⠀⠀⠀⢀⣾⡟⣿⠁⠀⠀⠀⠀⠀⠀⣿⣿⣇⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⢸⡟⠀⠀⠀⠀⠀⠀⠀⠀⠈⢷⣌⠿⣦⣀⠀⠀⠀⠀⣀⣀⣤⡾⠋⠀⠙⢿⣦⣀⠀⠀⠀⠀⠀⣀⣴⡿⠋⡼⠃⠀⠀⠀⠀⠀⠀⠀⣿⣿⣿⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⢸⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠹⠷⣌⠛⠿⠶⠶⠿⠿⠛⠉⠀⠀⠀⠀⠀⠉⠛⠻⠿⠷⠾⠿⠟⠋⠠⠞⠁⠀⠀⠀⠀⠀⠀⠀⢀⣿⣯⣿⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⢸⣿⣄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠉⠑⠒⠂⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠒⠒⠂⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣸⣿⣿⣿⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⢸⣿⣿⡄⠀⠀⠀⠀⣀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⠀⠀⠀⠀⣿⣿⣿⡇⠀⠀⠀⠀⠀⠀
⠀⠀⢀⣠⣶⡶⣾⣿⣿⣷⡄⠀⠀⠀⠻⠿⠲⠶⢦⣤⣤⣀⣀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣀⣤⣤⡴⠶⠖⢿⠟⠀⠀⠀⣤⣿⣿⣿⣥⣶⣶⣄⠀⠀⠀
⠀⠀⣸⣿⡿⣶⣿⣿⣿⣿⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠉⠉⠛⠛⠛⠒⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠛⠛⠛⠉⠉⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⣸⣿⣿⣿⣿⠾⢿⣿⡇⠀⠀
⠀⠀⣿⠏⠀⠀⠀⠈⢿⣿⣷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠶⢶⣤⣤⣄⣀⣀⡀⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢠⣿⣿⡿⠋⠀⠀⠀⠹⣷⠀⠀
⠀⣼⡏⠀⠀⠀⠀⠀⠈⢿⣿⣆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣴⣿⣿⡿⠁⠀⠀⠀⠀⠀⢹⡇⠀
⠀⣿⠁⠀⠀⠀⠀⠀⠀⠘⣿⣿⣆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣰⣿⣿⣿⠁⠀⠀⠀⠀⠀⠀⠈⣿⠀
⠀⣿⠀⢀⣰⣶⣄⠀⠀⠀⣿⡎⠻⣷⣄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⣿⣿⠟⢡⣿⠀⠀⠀⢀⣤⣄⠀⠀⣿⠀
⢸⡏⠀⠀⠉⣽⣿⡀⠀⠀⢸⣧⠀⠈⠻⣷⣦⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣴⣾⣿⡿⠋⠀⢸⡇⠀⠀⠀⢸⣏⠀⠀⠀⢻⡆
⢸⡇⠀⠀⠛⠻⣿⣇⠀⠀⢀⣿⠀⠀⠀⠈⠻⢿⣷⣦⣄⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣠⣴⣾⣿⡿⠟⠁⠀⠀⠀⣸⠃⠀⠀⢀⣿⣿⠃⠀⠀⢸⡇
⢸⡇⠀⠀⠶⣶⡾⠿⠶⣶⠾⠟⠀⠀⠀⠀⠀⠀⠈⠙⠻⢿⣿⣷⣶⣶⣤⣤⣀⣀⣀⣀⣀⣀⣤⣤⣴⣶⣾⣿⣿⣿⠿⠛⠉⠀⠀⠀⠀⠀⠀⠸⠷⢶⣶⡿⣋⣿⠂⠀⠀⣸⠇
⠀⣿⡄⠀⠀⣸⡷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠉⠉⠙⠛⠛⠛⠛⠛⠛⠙⠛⠛⠛⠛⠋⠉⠉⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⢻⣿⡀⠀⢀⣿⠁
⠀⠘⠿⠿⠟⠛⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠙⠿⠶⠿⠋⠀

**Comment by Alex-Hayeur** [comment]
lgtm 👍

File: .gitignore (hunk 1 above)
Comment (Alex-Hayeur) [reply .gitignore:15]: I know it's not husky's fault it's just funny to see you adding it.

**Review by andrewdaly124 — APPROVED** [review]
lovely

**Review by Alex-Hayeur — APPROVED** [review]

**Pull request merged**